R/test-threads.R

```diff
@@ -4,11 +4,11 @@
 #' @param .threads Integer vector of threads values to test.
 #' @param .bbi_args a named list.
 #' @param .max_eval Max number of iterations for NONMEM to run.
-#'         Will update `MAXITER` or `NITER` (whichever is specified) in generated models.
+#'         Will update `MAXITER`, `NITER`, and `NBURN` (whichever is specified/found) in generated models.
 #'         The best number for this is model-dependent. Typically, something between 10 and
 #'         100 is good, depending on how long each iteration takes. You want something that
 #'         will run for 3-5 minutes total. You can set this argument to `NULL` to run with the
-#'         same settings as the original model.
+#'         same settings as the original model. Any option set to `0` will not be overwritten.
 #' @param ... args passed through to submit_models()
 #'
 #' @importFrom checkmate assert_list assert_int
```

Comment:
Most of the `@importFrom` values for `test_threads` are no longer used within `test_threads` itself and should be moved to the new helper function instead.